CBLClient/Database.py

```diff
@@ -230,15 +230,20 @@ def create_bulk_docs(self, number, id_prefix, db, channels=None, generator=None,
                 doc_body = doc_generators.simple_user()
             elif generator == "complex_doc":
                 doc_body = doc_generators.complex_doc()
+            elif generator == "20mb":
+                doc_body = doc_generators.doc_20mb()
             else:
                 doc_body = doc_generators.simple()
 
             if channels is not None:
                 doc_body["channels"] = channels
 
             if attachments_generator:
-                types.verify_is_callable(attachments_generator)
-                attachments = attachments_generator()
+                if attachment_file_list is not None:
+                    attachments = attachments_generator(attachment_file_list)
```

Comment:
you are attaching one file, does it need to be sent as a list? Can you make it as one file ?

Reply:
this is function which can be used in different scenario. It is not designed just for one case, so we need to plan it on scalable base.